garak/command.py

```diff
@@ -255,3 +255,48 @@ def write_report_digest(report_filename, digest_filename):
     digest = report_digest.compile_digest(report_filename)
     with open(digest_filename, "w", encoding="utf-8") as f:
         f.write(digest)
+
+def detector_only_run():
```

Comment:
@jmartin-tech What do you think this is telling us about where responsibility for orchestrating runs lies? Is the existing `Harness` interface just too inflexible to make invoking novel things like `DetectorOnly` from `garak.cli`?